src/components/profile/PinList.jsx

```diff
@@ -1,14 +1,200 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import { default as icDrop } from '../../assets/icon_drop.svg';
+import DropBox from '../../components/common/DropBox';
+import { pinterestColors } from '../../styles/color';
+import BoardItem from '../common/BoardItem';
+import { boardData, board1_pinData, board2_pinData } from './testData.js';
+
+const dropBoxData = { text: '정렬 기준', options: ['알파벳순', '사용자 지정', '마지막 저장일'] };
+
+const test = [1, 1, 1, 1, 1];
 
 function PinList() {
-  return <StyledRoot>보드 영역 입니다</StyledRoot>;
+  const [openModal, setOpenModal] = useState(false);
+
+  const openSortingModal = () => {
+    let newModalState = !openModal;
+    setOpenModal(newModalState);
+  };
```

Comment:
```suggestion
  const openSortingModal = () => {
    setOpenModal(prev => !prev);
  };
```
굳이 로컬 변수 하나 더 만들지 않고 이렇게 써도 될 것 같아

Reply:
헉 그러게! 오케이오케이!